tex/gregoriotex.lua

```diff
@@ -587,6 +587,7 @@ end
 local function direct_gabc(gabc, header)
   tmpname = os.tmpname()
   local f = io.open(tmpname, 'w')
+  gabc = gabc:match('^()%s*$') and '' or gabc:match('^%s*(.*%S)')
```

Comment:
I'm having difficult time undestanding what this does... could you just add a short comment above?